config/core/roles/pingsource-mt-adapter-clusterrole.yaml

```
@@ -28,6 +28,18 @@ rules:
       - "get"
       - "list"
       - "watch"
+  - apiGroups:
+      - ""
+    resources:
+      - "serviceaccounts/token"
+    verbs:
+      - "get"
+      - "list"
+      - "create"
+      - "update"
+      - "delete"
+      - "patch"
+      - "watch"
```

```suggestion
      - "create"
```

I think we only need "create"